test/roby_app/ftp_server/test_server.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+require "minitest/autorun"
+
+require "ftpd"
+require "net/ftp"
+
+require "syskit/roby_app/ftp_server"
+
+class FtpServerTest < Minitest::Test
+    # auxiliary functions
+
```

Comment:
I prefer the spec-style organization of tests, i.e. using `describe` and `it`. I do not use the spec-style assertions (i.e. still using the `assert_*` and `refute_*` calls. Syskit has rather old code, which is still using a mixed style (or even the test style), but all new tests in here should be using describe/before/after/it

The reason is that `describe` blocks can be created recursively, while retaining the before/after blocks, which allows to keep the tests well organized.